modules/fedimint-wallet/src/lib.rs

```diff
@@ -359,6 +365,81 @@ impl ModuleGen for WalletGen {
     fn hash_client_module(&self, config: serde_json::Value) -> anyhow::Result<sha256::Hash> {
         serde_json::from_value::<WalletClientConfig>(config)?.consensus_hash()
     }
+
+    async fn dump_module_database(
+        &self,
+        dbtx: &mut DatabaseTransaction<'_>,
+        prefix_names: Vec<String>,
+    ) -> Box<dyn Iterator<Item = (String, Box<dyn erased_serde::Serialize + Send>)> + '_> {
+        let mut wallet: BTreeMap<String, Box<dyn erased_serde::Serialize + Send>> = BTreeMap::new();
+        for table in DbKeyPrefix::iter() {
+            filter_prefixes_by_name!(table, prefix_names);
```

Comment:
This macro expanding to something that alerts the control flow (`continue`) is very sneaky. IMO, the macro should contain only the check, `if matches_smth_smth!(...) { continue; }`.

Reply:
Macro removed

Reply:
Yeah, think I accidentally lost a commit. Will fix.